src/sentry/api/endpoints/group_autofix_setup_check.py

```diff
@@ -118,6 +121,23 @@ def get(self, request: Request, group: Group) -> Response:
                 "repos": repos,
             }
 
+        # Check the current user first using PromptsActivity
+        user_has_acknowledged = PromptsActivity.objects.filter(
+            user_id=request.user.id,
+            feature=feature_name,
+            organization_id=org.id,
+            project_id=0,
+        ).exists()
+
+        # If the user acknowledged, the org must have too. Otherwise, check the org.
+        org_has_acknowledged = user_has_acknowledged
```

Comment:
how will that be enforced? through the frontend only or is there another endpoint where these acknowledgements get set? Could a user potentially send an API request to acknowledge for themselves and bypass the org check this way?

(genuinely not sure, just asking)

Reply:
its just saving a query here. anyone in the org can acknowledge, if the user making this request has acknowledged then we know someone in the org has acknowledged